src/tests/common/element-snapshot-formatter.ts

```diff
@@ -77,7 +77,7 @@ export const CSS_MODULE_HASH_REPLACEMENT = '{{CSS_MODULE_HASH}}';
 // Our webpack config adds generated suffixes of form "--abc12" to the end of class names defined in
 // CSS. This normalizes them to avoid causing E2Es to fail for unrelated style changes.
 export function normalizeCssModuleClassName(className: string): string {
-    const cssModuleClassNameMatcher = /^([\w-]+--)[A-Za-z0-9+\/=-]{5}$/;
+    const cssModuleClassNameMatcher = /^([\w-]+--)[A-Za-z0-9+\/=-_]{5}$/;
```

Comment:
The hash from css-modules can produce '_' characters (https://tools.ietf.org/html/rfc4648#section-5), which we weren't covering before.

As per: https://nodejs.org/api/buffer.html#buffer_buffers_and_character_encodings